cpp/src/rolling/detail/range_window_bounds.hpp

```diff
@@ -69,6 +73,13 @@ struct range_type_impl<TimestampType, std::enable_if_t<cudf::is_timestamp<Timest
   using rep_type = typename type::rep;
 };
 
+template <typename FixedPointType>
+struct range_type_impl<FixedPointType,
+                       std::enable_if_t<cudf::is_fixed_point<FixedPointType>(), void>> {
```

Comment:
Zut alors! I have missed one!

Reply:
Actually, the one at Line#78 is not of the form `enable_if_t<_,void>* = nullptr`, so it can't be replaced with `CUDF_ENABLE_IF()`. :/

But I did indeed miss one at Line#113. That's been corrected now. Thank you!